src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLFeatureUtil.java

```diff
@@ -506,30 +504,27 @@ public static List<Link> alternateProjections(
             String base, String path, Map<String, Object> query) {
         ArrayList<Link> links = new ArrayList<>();
         Set<String> projections = TiledCRSConstants.tiledCRSBySrsName.keySet();
-        projections.forEach(
-                (String p) -> {
-                    Link l = new Link();
-                    TiledCRSParams projection = TiledCRSConstants.lookupTCRS(p);
-                    try {
-                        l.setProjection(ProjType.fromValue(projection.getName()));
-                    } catch (FactoryException e) {
-                        throw new ServiceException("Invalid TCRS name");
-                    }
-                    l.setRel(RelType.ALTERNATE);
-                    query.put("srsName", "MapML:" + projection.getName());
-                    HashMap<String, String> kvp = new HashMap<>(query.size());
-                    query.keySet()
-                            .forEach(
-                                    key -> {
-                                        kvp.put(key, query.getOrDefault(key, "").toString());
-                                    });
-                    l.setHref(
-                            ResponseUtils.urlDecode(
-                                    ResponseUtils.buildURL(
-                                            base, path, kvp, URLMangler.URLType.SERVICE)));
-                    links.add(l);
-                });
-
+        for (String proj : projections) {
+            Link l = new Link();
+            TiledCRSParams projection = TiledCRSConstants.lookupTCRSParams(proj);
+            boolean addMe = false;
+            l.setProjection(projection.getName());
+            addMe = true;
+
+            if (!addMe) continue;
```

Comment:
Uh? How does one even get "addMe" to be false there? Sounds like a debugging code bit left around.